### DISCUSSION TITLE
Windows paylaod not working please help.

### DISCUSSION THREAD

**Discussion by infosecZombie** [t3l3machus/Villain#36] [Q&A]
Hello there, I am getting this error while using the obfuscated payload for windows.

Payload command used:
`generate os=windows lhost=eth0 obfuscate`

**CMD output:**

> 'StArT-PR'OCESs'' is not recognized as an internal or external command, operable program or batch file.

**Powershell output:**

> This script contains malicious content and has been blocked by your antivirus software.

**Comment by t3l3machus** [comment]
You need to obfuscate manually. Check out for ideas: https://www.youtube.com/watch?v=FVbdZSGkzhs
